packages/react-native-editor/ios/Podfile

```diff
@@ -3,6 +3,8 @@ project 'GutenbergDemo.xcodeproj'
 require_relative '../../../node_modules/react-native/scripts/react_native_pods'
 require_relative '../../../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
+pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/WordPress-Aztec-iOS.git', :commit => 'a41d881f6a831fecef9e6400e69a64aafe80d9ac'
+
```

Comment:
Friendly reminder to remove this once the Aztec changes are merged and a new version is referenced in Gutenberg.